examples/openshift/openshift-values.yaml

```diff
@@ -46,6 +45,10 @@ spiffe-oidc-discovery-provider:
     runAsUser: null
     runAsGroup: null
     fsGroup: null
+  securityContext:
+    allowPrivilegeEscalation: true
+    runAsNonRoot: false
+    privileged: true
```

Comment:
Why did you need these bits? I was able to do it on openshift local without it. How does it break?

Reply:
You are correct. I just test it and removed it.